src/main/java/eu/eutampieri/catacombs/model/EntityKind.java

```diff
@@ -25,5 +25,13 @@
     /**
      * A generic item.
      */
-    ITEM;
+    ITEM,
+    /**
+     * A generic bullet
+     */
+    BULLET,
+    /**
+     * The boss's special bullet
+     */
+    BOSS_BULLET;
 }
```

Comment:
maybe we can rename the class to something else like Object type

Reply:
Aren't these entities (in the broader term)